libs/vm-modules/tests/unit/ml/ml_model_tests.cpp

```diff
@@ -848,4 +853,18 @@ TEST_F(VMModelTests, model_with_accuracy_metric)
   EXPECT_LE(metrics->elements.at(1), 1);
 }
 
+TEST_F(VMModelTests, DISABLED_model_sequential_flatten)
+{
+  static char const *SRC_METRIC = R"(
+        function main()
+          var model = Model("sequential");
+          model.add("flatten");
+          model.compile("scel", "adam", {"categorical accuracy"});
+        endfunction
+      )";
+
+  ASSERT_TRUE(toolkit.Compile(SRC_METRIC));
+  ASSERT_TRUE(toolkit.Run());
```

Comment:
you should extend this test (or write another one if you prefer), so that you actually pass in some data to the model, call model.predict, return the output tensor from the etch function, and compare the shape and values of the tensor with another 'ground truth' tensor.

Reply:
done